.github/workflows/nightly.yml

```diff
@@ -62,6 +63,7 @@ jobs:
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics --exclude shark.venv,lit.cfg.py 
     - name: Build and validate the IREE package
+      if: matrix.backend == IREE
```

Comment:
Do you need double curly braces ?

Reply:
Quotes were missing. YML format is funky!

Reply:
I thought we needed this kind: https://github.com/llvm/torch-mlir/blob/068eec41cb959278d269cd1d307cd41c9a4b4c0c/.github/workflows/buildAndTest.yml#L94

Reply:
Oh OK. I'll add that in while I fix the SHARK build and validate job